From 191c274b6024a3b3055200e909d6ad68872fa976 Mon Sep 17 00:00:00 2001
From: Sean Nyekjaer <sean@nyekjaer.dk>
Date: Sun, 25 Jun 2017 21:20:05 +0200
Subject: [PATCH 7/8] arm64: dts: marvell: Add microsd card definition for the
 ESPRESSObin

This defines and enables the microsd card on the
Marvell ESPRESSObin board.

Signed-off-by: Sean Nyekjaer <sean@nyekjaer.dk>
---
 arch/arm64/boot/dts/marvell/armada-3720-espressobin.dts | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-3720-espressobin.dts b/arch/arm64/boot/dts/marvell/armada-3720-espressobin.dts
index b1af3f988b29..409385c4195a 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-espressobin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-espressobin.dts
@@ -45,6 +45,7 @@
 
 /dts-v1/;
 
+#include <dt-bindings/gpio/gpio.h>
 #include "armada-372x.dtsi"
 
 / {
@@ -76,6 +77,16 @@
 	status = "okay";
 };
 
+/* microsd slot */
+&sdhci1 {
+	wp-inverted;
+	bus-width = <4>;
+	cd-gpios = <&gpionb 3 GPIO_ACTIVE_LOW>;
+	no-1-8-v;
+	marvell,pad-type = "sd";
+	status = "okay";
+};
+
 /* J7 */
 &usb3 {
 	status = "okay";
-- 
2.13.2

